libs/offline-tasks/src/main/java/org/opensearch/offline_tasks/TaskClient.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.offline_tasks;
```

Comment:
Do we follow underscoring multiword package names as a convention?
```suggestion
package org.opensearch.offlinetasks;
```

Reply:
I am not sure if there is a convention in opensearch for package naming. Java supported both. And i also see packages using both styles. 

with `_` -> `libs/secure-sm`
without `_` -> `libs/x-content`

Reply:
I am fine with either one. Tagging @sachinpkale  and @Bukhtawar  for preference